lib/hmmlearn/base.py

```diff
@@ -482,7 +478,7 @@ def fit(self, X, lengths=None):
 
         Parameters
         ----------
-        X : array-like, shape (n_samples, n_features)
+        X : array-like, shape (n_samples, n_features_)
```

Comment:
The docs don't use a trailing underscore here (the same comment applies throughout).

Reply:
Done in fit() now.